cfgov/jinja2/v1/_includes/organisms/snippet-list.html

```diff
@@ -55,7 +55,7 @@
     } ) }}
 
     {% set snippets = get_snippets(value.snippet_type) %}
-    {% set filtered_snippets = snippets.objects.filter_by_tags(value.tags) %}
+    {% set filtered_snippets = snippets.objects.filter_by_tags(value.tags) if snippets.objects.filter_by_tags else snippets.objects.all() %}
```

Comment:
For posterity (and as we discussed), it'd be better if the logic that defines the `snippets` variable was part of the block Python code. Unfortunately this would be a bigger change so this is a good way to implement this for now.